src/main/java/com/pnu/dev/shpaltaif/service/CategoryServiceImpl.java

```diff
@@ -54,6 +66,9 @@ public void create(CategoryDto categoryDto) {
                 .build();
 
         categoryRepository.save(category);
+
+        telegramBotUserService.addCategorySubscriptionForAllUsers(category);
+        telegramNotificationService.sendNotificationsOfNewCategory(category);
```

Comment:
I`m not sure, but maybe we should pass `categoryFromDb` as a parameter here

Reply:
It's ok, as long as we don't need to use id.